docs/project/library-servicing.md

```diff
@@ -16,6 +16,10 @@ If a library is packable (check for the `<IsPackable>true</IsPackable>` property
 
 When you make a change to a library & ship it during the servicing release, the `ServicingVersion` must be bumped. This property is found in the library's source project. It's also possible that the property is not in that file, in which case you'll need to add it to the library's source project and set it to 1. If the property is already present in your library's source project, just increment the servicing version by 1.
 
+## Optionally ensure all up-stack packages are also produced
+
+If you with to ensure that every package that references a serviced package is also serviced itself, you can enable validation by setting `ServiceTransitiveDependencies` to true.  When doing this then building the repo, eg: `build libs -allConfigurations` you'll see errors from any project that didn't enable servicing.
```

Comment:
typo: "with" -> "wish".

Who are the expected users of `ServiceTransitiveDependencies=true` ? Assuming incremental servicing, wouldn't we want to use this for our own OOB packages?

Reply:
The expected users would be those building the entire repo and wanting all up-stack packages to be updated.   It's currently not the default behavior, but it's useful in this PR since to see the impact of removing dependencies we need to update all packages that will be impacted by those removals.

I might refine this to also allow it to be specified optionally in the serviced package too.  That would let folks do this more granularly if they wanted.  For now, this is primarily an off-by-default experiment to see if we might wish to use this workflow in the future.